keylime/keylime_logging.py

```diff
@@ -10,26 +10,38 @@
 from keylime import config
 
 
+LOG_TO_FILE = ['registrar', 'provider_registrar', 'cloudverifier']
+LOG_TO_STREAM = ['tenant_webapp']
+LOGDIR = os.getenv('KEYLIME_LOGDIR', '/var/log/keylime')
+# not clear that this works right.  console logging may not work
+if not config.REQUIRE_ROOT:
+    LOGSTREAM = './keylime-stream.log'
+else:
+    LOGSTREAM = LOGDIR + '/keylime-stream.log'
+
+logging.config.fileConfig(config.CONFIG_FILE)
+
+
+def set_log_func(loglevel, logger):
+    log_levels = {
```

Comment:
It's probably not a huge performance difference, but it would actually be better if this `log_levels` structure was created outside of this function instead of every time the function is called as a local structure that then also has to be torn down.

Reply:
makes sense

Reply:
@mpeters I just have one doubt on this:
considering that the structure depends on logger, this is not a static structure, so it needs to be generated at least once
I'm not sure I see a different way to do it, unless changing other modules

Reply:
Ah you're correct. So going back to the original code, I know it might not look as nice, but if/then statements are really fast to execute. Either way it's not a big deal performance wise, but it should be faster to just take the old code and factor it out into a function vs creating a onetime use structure and throwing it away on every call.

Reply:
yeah, I was afraid of this, I did a quick test and the dictionary approach is way slower
I'm going to revert to if/else and move that to a function